src/main/user/user-request-authorizer.ts

```diff
@@ -1,24 +1,33 @@
 import { getTokenDetails } from "./user-resolver";
+import { isUserAuthorized } from "../role/roles-based-authorizer";
 
 export const ERROR_TOKEN_MISSING = {
   error: "Bearer token missing",
-  message: "You are not authorized to access this resource",
+  message: "You are not authorised to access this resource",
   status: 401,
 };
 export const ERROR_UNAUTHORISED_ROLE = {
   error: "Unauthorised role",
-  message: "You are not authorized to access this resource",
+  message: "You are not authorised to access this resource",
   status: 403,
 };
 export const ERROR_UNAUTHORISED_USER_ID = {
   error: "Unauthorised user",
-  message: "You are not authorized to access this resource",
+  message: "You are not authorised to access this resource",
   status: 403,
 };
 
 export const COOKIE_ACCESS_TOKEN = "accessToken";
 export const AUTHORIZATION = "Authorization";
 
+const authorizeRoles = (user) => new Promise((resolve, reject) => {
+  if (!isUserAuthorized(user)) {
+    reject(ERROR_UNAUTHORISED_ROLE);
+  } else {
+    resolve();
+  }
+});
+
 export const authorise = (request) => {
   let user;
   const bearerToken = request.get(AUTHORIZATION) || (request.cookies ? request.cookies[COOKIE_ACCESS_TOKEN] : null);
```

Comment:
This is old code `AUTHORIZATION` and now we also have `ERROR_UNAUTHORISED_ROLE`

Reply:
I'll correct, so the variable name is Americanised (or Americanized :))